core/src/main/java/org/elasticsearch/index/query/GeoDistanceQueryBuilder.java

```diff
@@ -98,7 +89,6 @@ public GeoDistanceQueryBuilder(StreamInput in) throws IOException {
         distance = in.readDouble();
         validationMethod = GeoValidationMethod.readFromStream(in);
         center = in.readGeoPoint();
-        geoDistance = GeoDistance.readFromStream(in);
```

Comment:
With maintaining the ability for rolling upgrades from 5.last to 6.0, do we need to maintain bwc logic in here (at least to skip over?)?